earth_enterprise/SConstruct

```diff
@@ -154,14 +154,14 @@ num_jobs = GetOption('num_jobs')
 
 # Build Fusion and Server
 build_cmds = {'build': [
-    'cd src; python2.7 /usr/bin/scons -j%d %s third_party' % (num_jobs, arg_vars),
-    'cd src; python2.7 /usr/bin/scons -j%d %s' % (num_jobs, arg_vars)
```

Comment:
python2.7 is needed for scons now for el6.

Reply:
Probably just needs to be updated to the latest master since this is newer. I see other missing changes as well.